lib/src/js/compile.dart

```diff
@@ -182,6 +186,16 @@ OutputStyle _parseOutputStyle(String? style) => switch (style) {
 /// Converts [importer] into an [AsyncImporter] that can be used with
 /// [compileAsync] or [compileStringAsync].
 AsyncImporter _parseAsyncImporter(Object? importer) {
+  if (importer is JSNodePackageImporter) {
+    if (isBrowser) throwMissingFileSystem();
+    var entryPointPath = importer.entryPointPath != null
+        ? p.join(p.current, importer.entryPointPath)
```

Comment:
This is equivalent to `p.absolute(importer.entryPointPath)`, but why is making this absolute necessary at all?

Reply:
The entryPointPath may be relative or absolute, and the spec specifies that relative paths will be resolved relative to the CWD. The NodePackageImporter requires an associated absolute path. 

I realized I could move this into the constructor as well, and did that in 125acf351141814789ab7ae3797a48226dd7137f.